### PR TITLE
Reader Revenue message on AU front

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
Like we did for the US end of year campaign, we are adding a new component in for the AU campaign. Similarly to the previous implementation for the US, this renders in the Fronts container so we will need to create a new container in the fronts tool with the name '`AU end-of-year 2024`' so it renders.

## Why?
Explanation from Tom  (from [this PR](https://github.com/guardian/dotcom-rendering/pull/12593))
_In the past we've used "Thrashers" to deploy this type of message to the fronts. However, thrashers are not suitable for rich/interactive components like this. This component needs to render a ticker (based on a fetch) and interactive choice cards.
It's now much easier to build this type of thing in DCR as islands, where the relevant source components etc exist. It's also easier to line it up with the columns of the fronts._

## Screenshots
image

[before]: https://example.com/before.png
[after]: https://example.com/after.png

### CODE REVIEW

**Comment by charleycampbell** [comment]
This needs signed off by the AU team

**Comment by charleycampbell** [comment]
> This needs signed off by the AU team

This has been signed off by the AU team

File: dotcom-rendering/src/components/AusEoy2024Wrapper.importable.tsx [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+	}
+	return;
+};
+
+const choiceCardAmounts: SelectedAmountsVariant = {
+	testName: 'au-eoy-front-amounts',
+	variantName: 'CONTROL',
+	defaultContributionType: 'MONTHLY',
+	displayContributionType: ['ONE_OFF', 'MONTHLY', 'ANNUAL'],
+	amountsCardData: {
```
Comment (tomrf1) [inline dotcom-rendering/src/components/AusEoy2024Wrapper.importable.tsx:150]: are these amounts taken from the US version? These should probably match the AU amounts in the RRCP amounts tool

Comment (charleycampbell) [reply dotcom-rendering/src/components/AusEoy2024Wrapper.importable.tsx:150]: oh good point!!

Comment (charleycampbell) [reply dotcom-rendering/src/components/AusEoy2024Wrapper.importable.tsx:150]: updated as per the RRCP amount tool

**Review by tomrf1 — APPROVED** [review]
👍

**Review by tomrf1 — APPROVED** [review]

**Pull request merged**